plugin-build/gradle.properties

```diff
@@ -0,0 +1,8 @@
+ID=io.sentry.kotlin.multiplatform.gradle.plugin
```

Comment:
nit: for android we just use `io.sentry.android.gradle`, so maybe we could drop the last `.plugin` part here as well